lib/sparse/src/index/compressed_posting_list.rs

```diff
@@ -498,8 +699,9 @@ mod tests {
     fn test_iter() {
         for case in cases() {
             let list = CompressedPostingList::<f32>::from(case.clone());
+            let hw_counter = HardwareCounterCell::disposable();
```

Comment:
`::disposable()` is for disposable HwCounter in the code. For tests we have `::new()`.
Also in other places.

```suggestion
            let hw_counter = HardwareCounterCell::new();
```